src/app/services/item.service.ts

```diff
@@ -46,7 +46,7 @@ export class ItemService {
         if (!subscriptionData.data.tasks.task) {
           return prev;
         }
-        if (subscriptionData.data.tasks.mutation === MutationType.CREATED) {
+        if (subscriptionData.data.tasks.action === MutationType.CREATED) {
```

Comment:
Wouldn't be better to use a `switch` here?

Reply:
We have rule that will transform nested if statements if there is more than 3 occurrences. 
Here we have 2 so we are fine with if.

Reply:
Then I believe we should have at least an `else if`